emitter/README.md

```diff
@@ -23,29 +39,28 @@ kubectl apply -f broker.yaml
 kubectl apply -f service.yaml
 ```
 
-List the services to obtain the IP address of the Emitter load balancer.
-
-```
- % kubectl --namespace openfaas get service emitter
- NAME      TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S) AGE
- emitter   ClusterIP   10.111.110.157    <pending>  8080:30790/TCP,8443:30705/TCP   8m40s
-```
+## Check the deployment
 
 Check the logs of the service:
 
 ```
 kubectl logs statefulset.apps/broker -n openfaas
 ```
 
-Port-forward the Emitter's UI so that you can generate a channel key.
 
-```sh
-kubectl port-forward -n openfaas svc emitter 8080:8080 &
+You can now visit the `emitter` subdomain of your domain to view the Emitter
+keygen UI, i.e. `https://emitter.example.com/keygen`.
+
+Alternatively, if you've not setup a domain name for the workshop, then you can
+view the Metabase UI using Kubernetes port forwarding.
+
 ```
+export POD_NAME=$(kubectl get pods --namespace default -l "app=emitter,release=metabase" -o jsonpath="{.items[0].metadata.name}")
```

Comment:
Isn't this overly-complicated?

See my examples:

```
kubectl port-forward deploy/metabase -n default 8080:80
```

I would also avoid port 8080 locally since that's used by OpenFaaS already when deploying.

8081 by the emitter, try 8082